libnmstate/nm/sriov.py

```diff
@@ -72,8 +73,14 @@ def get_info(device):
     )
 
     if sriov_setting:
```

Comment:
The case where a user changes this value on the kernel directly (e.g. though sysfs) is valid. The report should show this, but if the user never configured a profile for the device, this solution will not report it.

Reply:
Fixed! Thanks. Please see the comment below.